src/mode.js

```diff
@@ -130,6 +122,25 @@ function getRtvVersion(win) {
   return `01${internalRuntimeVersion()}`;
 }
 
+/**
+ * Triggers validation or enable pub level logging. Validation can be
+ * bypassed via #validate=0.
+ * Note that AMP_DEV_MODE flag is used for testing purposes.
+ * Uses Array.indexOf instead of Array.includes because of #24219
+ * @param {!Window} win
+ * @return {boolean}
+ */
+export function isModeDevelopment(win) {
+  const hashQuery = parseQueryString_(
+    win.location['originalHash'] || win.location.hash
+  );
+  return !!(
+    ['1', 'actions', 'amp', 'amp4ads', 'amp4email'].indexOf(
+      hashQuery['development']
+    ) >= 0 || win.AMP_DEV_MODE
+  );
```

Comment:
```suggestion
  return !!(
    ['1', 'actions', 'amp', 'amp4ads', 'amp4email'].includes(
      hashQuery['development']
    ) || win.AMP_DEV_MODE
  );
```
We polyfill this in IE anyway